### ISSUE TITLE
ModuleNotFoundError: No module named 'werkzeug.contrib'

### ISSUE THREAD

**Issue by thinkh** [phovea/phovea_server#105]
* Release number or git hash: 4.0.1 and develop
* Web browser version and OS: Python 3
* Environment (local or deployed): both

### Steps to reproduce

1. Rebuild the docker container with newest dependencies `docker-compose build --force api`
2. `docker-compose up`

### Observed behavior

```
 init config
configuration file: /phovea.config/config.json
09:00:58 INFO phovea_server.launch: enabling production mode
09:00:59 INFO phovea_server._plugin_parser: looking for production neighbors
09:00:59 INFO phovea_server._plugin_parser: discovered 13 plugins: ['phovea_clue', 'phovea_data_mongo', 'phovea_data_redis', 'phovea_security_flask', 'phovea_server', 'tdp_bi_bioinfodb', 'tdp_core', 'tdp_publicdb', 'tdp_uploaded_data']
09:00:59 INFO phovea_server._plugin_parser: importing module: dtiles
09:00:59 INFO phovea_server._plugin_parser: importing module: phovea_clue
09:00:59 INFO phovea_server._plugin_parser: importing module: phovea_data_mongo
09:00:59 INFO phovea_server._plugin_parser: importing module: phovea_data_redis
09:00:59 INFO phovea_server._plugin_parser: importing module: phovea_security_flask
09:00:59 INFO phovea_server._plugin_parser: importing module: phovea_security_store_ldap
09:00:59 INFO phovea_server._plugin_parser: importing module: phovea_server
09:00:59 INFO phovea_server._plugin_parser: importing module: tdp_core
09:00:59 INFO phovea_server.launch: add command ldap
09:00:59 INFO phovea_server.launch: add command instance to parser
09:00:59 INFO phovea_server.launch: add command api
09:00:59 INFO phovea_server.launch: loading and initializing the command: api
09:00:59 INFO phovea_server.plugin: importing phovea_server.server
Traceback (most recent call last):
  File "launch.py", line 2, in <module>
    launch.run()
  File "/phovea/phovea_server/launch.py", line 198, in run
    main = args.launcher(args)  # execute the launcher function, which returns another function
  File "/phovea/phovea_server/server.py", line 205, in _launcher
    application = create_application()
  File "/phovea/phovea_server/server.py", line 171, in create_application
    from werkzeug.contrib.fixers import ProxyFix
ModuleNotFoundError: No module named 'werkzeug.contrib' 
```

### Expected behavior

* The server should run again

**Comment by thinkh** [comment]
The pip package [_Werkzeug_ was released yesterday in v1.0.0](https://pypi.org/project/Werkzeug/#history).

![grafik](https://user-images.githubusercontent.com/5851088/74017321-5a77a780-4994-11ea-9690-1b6494a3ddc2.png)

Looking at the CircleCI build I can confirm the version change.

**Yesterdays nightly build**

```
Collecting Werkzeug>=0.15
  Downloading Werkzeug-0.16.1-py2.py3-none-any.whl (327 kB)
```

**Todays nightly build** 

```
Collecting Werkzeug>=0.15
  Downloading Werkzeug-1.0.0-py2.py3-none-any.whl (298 kB)
```

We need to further investigate how to handle this dependency, since [_Werkzeug_ is **not** a direct dependency from phovea_server](https://github.com/phovea/phovea_server/blob/develop/requirements.txt).

**Comment by thinkh** [comment]
Some imports in our phovea_server are wrong, which got removed with Werkzeug v1.0.0.

[Werkzeug changelog v1.0.0](https://github.com/pallets/werkzeug/blob/dfde671ef969e27c7b14bd464688c009b34a7d2b/CHANGES.rst#version-100)

-   Drop support for Python 3.4. (:issue:`1478`)
-   Remove code that issued deprecation warnings in version 0.15. (:issue:`1477`)
-   Remove most top-level attributes provided by the ``werkzeug`` module in favor of direct imports. For example, instead of ``import werkzeug; werkzeug.url_quote``, do ``from werkzeug.urls import url_quote``. Install version 0.16 first to see deprecation warnings while upgrading. :issue:`2`, :pr:`1640`

[Werkzeug changelog v0.15.0](https://github.com/pallets/werkzeug/blob/dfde671ef969e27c7b14bd464688c009b34a7d2b/CHANGES.rst#version-0150)

* The contrib modules are deprecated and will either be moved into werkzeug core or removed completely in version 1.0. Some modules that already issued deprecation warnings have been removed. Be sure to run or test your code with python -W default::DeprecationWarning to catch any deprecated code you're using.
* `cache` has been extracted into a separate project, [cachelib](https://github.com/pallets/cachelib). The version in Werkzeug is deprecated.
* Middleware from werkzeug.wsgi has moved to separate modules under `werkzeug.middleware`, along with the middleware moved from `werkzeug.contrib`. The old `werkzeug.wsgi` imports are deprecated and will be removed in version 1.0.
